flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMapState.java

```diff
@@ -227,6 +220,7 @@ public void migrateSerializedValue(
 		}
 	}
 
+	@Nonnull
```

Comment:
As the [code stye](https://flink.apache.org/contributing/code-style-and-quality-common.html#nullability-of-the-mutable-parts) said , we can get rid of the annotation.
```
Note: This means that @Nonnull annotations are usually not necessary, but can be used in certain cases to override a previous annotation, or to point non-nullability out in a context where one would expect a nullable value.
```